### ISSUE TITLE
Debugger refers to channels numbers starting with 1 in some instances, starting with 0 in others.

### ISSUE THREAD

**Issue by Millennium-Eagle** [sauraen/seq64#21]
![Something is wrong with this picture.](https://user-images.githubusercontent.com/17091484/135192183-80ef30d1-db49-4eb1-a968-521842ad8ca4.PNG)

I double checked after seeing this and in the first instance, when checking short notes, they are addressing the channels as 1-16. When listing empty channels, they address them as 0-15. These and other times channel numbers are referred to should be checked for consistency. Ideally, there should be an option to choose between 0-15 and 1-16 depending on what editor you use, or at least a warning somewhere that this program only uses one of those methods.

**Comment by sauraen** [comment]
You are correct. The inconsistency problem is not just about the debug messages though. MIDI channel numbers are officially 1-16, and the Juce API functions for MIDI parsing etc. use 1-16, whereas Music Macro Language / Audioseq use 0-15. SEQ64 does convert properly (as far as I know!), but sometimes the debug messages are on one side of this conversion and sometimes on the other. Ideally you are right, it should be consistent. I will take a look at this at some point.
